Sources/SwiftGen/Config/Config.swift

```diff
@@ -27,7 +28,20 @@ extension Config {
     if !file.exists {
       throw Config.Error.pathNotFound(path: file)
     }
-    let anyConfig = try YAML.read(path: file)
+    
+    // Read contents and replace env variables
+    var contents: String = try file.read()
+
+    ProcessInfo.processInfo.environment.keys.forEach { key in
```

Comment:
I've always found `forEach` a bit weird, especially if you need the value later:
```suggestion
    for (key, value) in ProcessInfo.processInfo.environment {
```